### DISCUSSION TITLE
Main ("Cameras") page shows still images of cameras

### DISCUSSION THREAD

**Discussion by zoltanszalontay** [blakeblackshear/frigate#2520] [Ask A Question] [answered]
Dear all, a newbie question. 

Everything is working fine in my installation (live view, recording, object detection, mqtt alerting etc.), but the starting webpage ("Cameras") shows still images.

If I switch to another page back and forth, like Birdseye, the timestamp generated by the cameras change.

I have never seen the first page in real life so I do not know if it should be a live video or not. If I click on any of the camera images, the dedicated camera page show live video. It is only the main page that is static. 

Thanks in advance,
Z

**Comment by blakeblackshear** [comment, marked as answer]
This is expected. The cameras page is not intended to be a live view.

**Reply by zoltanszalontay** [reply]
Well, I guess I should stop debugging config.yml then. Finally, it's Christmas.

Thanks for your help and best wishes!
Zoltán
